src/_sass/base/_variables.scss

```diff
@@ -1,16 +1,18 @@
 // Colors
-$flutter-color-teal: #60caf6;
-$flutter-color-blue-500: #1389FD;
-$flutter-color-dark-blue: #075b9a;
-$flutter-color-fuchsia: #FF2B5B;
+$flutter-color-blue: #043875;
+$flutter-color-teal: #158477;
+$flutter-color-blue-on-dark: #B8EAFE;
+$flutter-color-blue-500: #0468D7;
+$flutter-color-dark-blue: #042B59;
+$flutter-color-fuchsia: #E64637;
 $flutter-color-light-blue: #E7F8FF;
 $flutter-dark-blue-texture: #232C33;
 $site-color-black: #000;
 $site-color-white: #FFF;
 $site-color-codeblock-bg: #F8F9FA;
-$site-color-light-grey: #D8D8D8;
+$site-color-light-grey: #F1F3F4;
```

Comment:
This variable is primarily used for the distinguishing lines around the left sidenav, but this new color is too light grey, and very hard to distinguish from the background. Could you adjust it to the slightly darker `#DADCE0` grey? Or introduce a new variable that uses that and adjust the various uses of this variable to use the new one? Thanks!


<br>
<img width="345" alt="Example of too light distinguishing lines" src="https://github.com/flutter/website/assets/18372958/a5264cc5-65f4-481d-a7b6-d53784b7e2e2">
<br>

```suggestion
$site-color-light-grey: #DADCE0;
```